server/text_generation_server/utils/merges/strategies.py

```diff
@@ -0,0 +1,223 @@
+import copy
```

Comment:
Probably worth adding a header here too, since it's mostly from https://github.com/predibase/lorax/blob/49bb52f357863974be100da19cc947f504c18d77/server/lorax_server/utils/merges/strategies.py#L43 ?

Reply:
great catch. updated